nats-concepts/what-is-nats/readme.md

```diff
@@ -1,41 +1,44 @@
 # What is NATS
 
+Software applications and services need to exchange data. NATS is an infrastructure that allows such data exchange, segmented in the form of messages. We call this a "**message oriented middleware**".
 
-NATS is what is called "message oriented middleware", meaning that it is a software infrastructure providing the exchange of data that is segmented into messages among computer applications and services.
+With NATS, application developers can:  
 
-NATS allows Application developers to easily and simply create distributed scalable client-server applications and generally store and distribute in real-time data between applications across different environments, languages, cloud providers and on-premise systems
+- Effortlessly build distributed and scalable client-server applications.
 
-### NATS Client Applications
+- Store and distribute data in realtime in a general manner. This can flexibly be achieved across various environments, languages, cloud providers and on-premises systems.
 
-Developers use one of the NATS client libraries in their application code to allow them to publish, subscribe, request and reply between instances of the application or between completely separate applications. Those applications are generally referred to as 'client Applications' or sometimes just as 'clients' (as they are clients from the NATS servers' point of view) throughout this manual.
+### NATS with Client Applications
```

Comment:
I would keep it as it was (no 'with'), NATS has two parts: the service infrastructure and the clients applications one doesn't make sense without the other, so you always have client applications and service infrastructure, so I would keep the two headings "NATS Client Applications" and "NATS Service Infrastructure" (rather than NATS with or NATS as)